tap_mysql/connection.py

```diff
@@ -16,6 +16,27 @@
 # We need to hold onto this for self-signed SSL
 match_hostname = ssl.match_hostname
 
+def get_request_timeout():
+    args = singer.utils.parse_args([])
+    # get the value of request timeout from config
+    config_request_timeout = args.config.get("request_timeout")
+
+    # return default value if timeout from config is none or empty
+    if not config_request_timeout:
+        return READ_TIMEOUT_SECONDS
+
+    if isinstance(config_request_timeout, int): # pylint: disable=no-else-return
+        # return value from config
+        return config_request_timeout
+    elif isinstance(config_request_timeout, str) and config_request_timeout.isdigit():
```

Comment:
@namrata270998 Why is this check not similar to other taps where we check that, `elif config_request_timeout and float(config_request_timeout)`?

Reply:
MySQL does not support float values for `net_read_timeout` variable. Document: https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_net_read_timeout. Hence only accepted int and int values in string
Added the same comment at line no. 35